core/sdk/src/api/firebolt/fb_lifecycle.rs

```diff
@@ -36,6 +38,19 @@ pub enum LifecycleState {
     Suspended,
 }
 
+impl From<&LifecycleState> for AppLifecycleState {
```

Comment:
If these are 1:1 mapping, why do we need `AppLifecycleState` at all , can't we just use `LifecycleState` where `AppLifecycleState` is used?

Reply:
AppLifecycleState is the Metrics LifecycleState they act a little differently compared to the regular app lifecycle state with apps launched with an intent of inactive: true. This change captures those use cases and makes the adjustments for those app launches.